src/sentry/static/sentry/app/components/menuItem.tsx

```diff
@@ -29,11 +29,6 @@ class MenuItem extends React.Component<Props> {
     eventKey: PropTypes.any,
     isActive: PropTypes.bool,
     noAnchor: PropTypes.bool,
-    // basic link
-    href: PropTypes.string,
-    // router link
-    to: PropTypes.string,
```

Comment:
Can we move these changes to a separate PR? I think we should focus on just `Link`